procedures/unit-testing-autorun.ipf

```diff
@@ -109,7 +110,13 @@ Function/S GetBaseFilename()
 	if(strlen(baseFilename))
 		return baseFilename
 	endif
-	sprintf baseFilename, "%s_%s_%s", IgorInfo(1), Secs2Date(DateTime, -2), ReplaceString(":", Secs2Time(DateTime, 3), "-")
+
+	if (SVAR_EXISTS(baseFilenameOverwrite))
```

Comment:
I think the approach to remove the global string when the option is not set has the disadvantage that for that case the full current state of the IUTF is not visible in its datafolder.

How about creating baseFilenameOverwrite always and use an empty string as disabled. You can check that with UTF_Utils#IsEmpty here.

btw: The there is a space here after the if and the function used here is SVAR_Exists.